breadbox/breadbox/api/datasets.py

```diff
@@ -456,6 +459,53 @@ def get_dimensions(
     return search_index_entries
 
 
+@router.post(
+    "/dimension/data/",
```

Comment:
I was on the fence about whether to make this endpoint a `POST` or `GET` method. The reason I chose `POST` here is because I thought it would be nice to keep our requests as close to a [SliceQuery](https://docs.google.com/document/d/175BBPZkNlMuvk_F-xYLs0xDphRZP66JCesUi7RWZECg/edit) JSON object as possible for consistency. 

With a `POST` method, you can use a request body like:
```
{
  "dataset_id": "string",
  "identifier": "string",
  "identifier_type": "feature_id"
}
```

Instead of needing to put the parameters in the URL. Having the request body (with no need for URL encodings), also makes things much more human-readable and therefor easier to debug in the browser (in my mind).

Reply:
I agree with this logic. I wish `GET`s could have json bodies, but alas they can't and thus `POST` is the way to go.